background/ncn_kdump.md

```diff
@@ -101,12 +101,33 @@ to be installed; the `crash` command can not thoroughly analyze a dump without t
 
         1. Install from Artifactory.
 
+            > ***NOTE*** CSM does NOT support the use of proxy servers for anything other than downloading artifacts from external endpoints.
+            Using **http_proxy** or **https_proxy** in any way other than the following examples will cause many failures in subsequent steps.
+
+            Without proxy:
+
             ```bash
             zypper ar https://$ARTIFACTORY_USER:$ARTIFACTORY_TOKEN@artifactory.algol60.net/artifactory/sles-mirror/Updates/SLE-Module-Basesystem/15-SP3/x86_64/update_debug/ temp-debug
             . /srv/cray/scripts/metal/dracut-lib.sh
             zypper --plus-content debug in -y kernel-default-debuginfo=${KVER%-default}
             ```
 
+            With https proxy:
+
+            ```bash
+            https_proxy=https://example.proxy.net:443 zypper ar https://$ARTIFACTORY_USER:$ARTIFACTORY_TOKEN@artifactory.algol60.net/artifactory/sles-mirror/Updates/SLE-Module-Basesystem/15-SP3/x86_64/update_debug/ temp-debug
+            . /srv/cray/scripts/metal/dracut-lib.sh
+            zypper --plus-content debug in -y kernel-default-debuginfo=${KVER%-default}
+            ```
+
+            With http proxy:
```

Comment:
Do we need the http proxy section here?   We are giving them a specific URL for this artifact and it is an HTTPS URL.  So I think the http_proxy would just be ignored in this case.